template/en/default/account/prefs/mfa.html.tmpl

```diff
@@ -61,7 +72,12 @@
     <input type="hidden" name="mfa_action" id="mfa-action" value="disable">
 
     <div class="mfa-buttons">
-      [% IF user.mfa != 'Duo' || (user.mfa == 'Duo' && !user.in_duo_required_group) %]
+      [% IF user.mfa == 'Duo' && user.in_duo_required_group %]
+        <p class="mfa-warning-msg">
+          You are in a group requiring Duo Security for two-factor authentication and 
+          therefore it cannot be disabled. Talk to admin if you need assistance.
```

Comment:
```suggestion
          therefore it cannot be disabled. Talk to an admin if you need assistance.
```

Also, is it worth calling out where to go to contact an admin? Since this is employee only it could just be "come to #bmo on Slack".

Reply:
Good idea. I will change that.